.github/scripts/generate_ci_workflows.py

```diff
@@ -172,6 +173,11 @@ def generate_workflow_file(
         test_runner_type=LINUX_CUDA_TEST_RUNNER,
         num_test_shards=2,
     ),
+    PyTorchLinuxWorkflow(
+        build_environment="pytorch-linux-bionic-rocm4.2-py3.6",
+        docker_image_base=f"{DOCKER_REGISTRY}/pytorch/pytorch-linux-bionic-rocm4.2-py3.6",
+        test_runner_type=LINUX_ROCM_TEST_RUNNER,
+    ),
```

Comment:
@seemethere Can we use the `num_test_shards=2` on ROCm CI as well, since we currently do that on the Jenkins CI?